test/jtreg-ext/requires/VMProps.java

```diff
@@ -252,6 +253,13 @@ protected String vmDebug() {
         }
     }
 
+    /**
+     * @return true if VM has experimental options unlocked
+     */
+    protected String vmUnlockExperimentalOptions() {
+        return "" + WB.getBooleanVMFlag("UnlockExperimentalVMOptions");
+    }
+
```

Comment:
I think an alternative here could be to just add UnlockExperimentalVMOptions to the list in vmOptFinalFlags and then use `!vm.opt.final.UnlockExperimentalVMOptions`. This is what we typically do when we want to filter on the final value of a specified flag.

Reply:
Ah that's much easier.  I saw that there were several patterns in this file, and that approach sipped my radar.  Will update.